tests/test_clpir.py

```diff
@@ -315,5 +315,434 @@ def test_encoding_methods_consistency(self) -> None:
         self.assertEqual(encoded_message_and_ts_delta, encoded_message + encoded_ts_delta)
 
 
+class TestCaseQueryBase(unittest.TestCase):
+    """
+    Base class for Query related testing.
+
+    Helper functions should be defined in this class. This base class shouldn't
+    be launched during the testing.
```

Comment:
```suggestion
@unittest.skip("Base class should not be used during testing.")
class TestCaseQueryBase(unittest.TestCase):
    """
    Base class for Query related testing.

    Helper functions should be defined in this class. 
```

Although currently there are no actual tests to run in this base class, it is better to explicitly guard against this.